### PR TITLE
[AWS] Update ECS version, add event.original options and preparing for package GA

### PR DESCRIPTION
**Pull request by P1llus**
## What does this PR do?

Updates package ECS version to 1.10.
Sync module changes to packages if any.
Adds Preserve Raw event functionality if not already exists.
Adds pipeline tests if missing.
Adds missing tags + moving tags to advanced settings.
Adds possibility to add custom processors.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

## Related issues

- Relates #994

### CODE REVIEW

File: packages/aws/data_stream/ec2_metrics/fields/fields.yml [hunk 1]
```diff
@@ -159,3 +159,9 @@
           type: integer
           description: |
             The number of threads per CPU core.
+    - name: cloudwatch
```
Comment (kaiyan-sheng) [inline packages/aws/data_stream/ec2_metrics/fields/fields.yml:162]: We might be missing this field in beats too... thanks for catching it!

**Review by kaiyan-sheng — APPROVED** [review]

File: packages/aws/data_stream/cloudtrail/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -6,15 +6,60 @@ processors:
...
+      target_field: event.original
+      ignore_missing: true
+      if: ctx._temp_?.json?.result?._raw != null
   - json:
-      field: "event.original"
-      target_field: "json"
+      field: event.original
+      target_field: json
+      if: ctx._temp_?.json?.result?._raw != null
+  - rename:
```
Comment (leehinman) [inline packages/aws/data_stream/cloudtrail/elasticsearch/ingest_pipeline/default.yml:42]: I might be wrong, but I don't think result.host or result.source make sense for CloudTrail.

**Review by leehinman — CHANGES_REQUESTED** [review]

File: packages/aws/data_stream/cloudtrail/elasticsearch/ingest_pipeline/default.yml (hunk 2 above)
Comment (P1llus) [reply packages/aws/data_stream/cloudtrail/elasticsearch/ingest_pipeline/default.yml:42]: Are they not the ones mapped with splunk? That part is inside the splunk specific parsing, and should just be a conversion from the edge processor used for splunk?

Comment (leehinman) [reply packages/aws/data_stream/cloudtrail/elasticsearch/ingest_pipeline/default.yml:42]: the original Splunk cloudtrail edge processing didn't copy `result.host` or `result.source` also they weren't included in the fingerprint.

pretty sure the data wasn't valuable.  `result.host` would be the Splunk machine that downloaded the data and `results.source` should be the S3 object which we already have.

Comment (P1llus) [reply packages/aws/data_stream/cloudtrail/elasticsearch/ingest_pipeline/default.yml:42]: Ah I must have gotten it from another splunk source, sorry about that, will modify it tomorrow then :) Did you find anything else or was this the only part @leehinman ?

Comment (leehinman) [reply packages/aws/data_stream/cloudtrail/elasticsearch/ingest_pipeline/default.yml:42]: everything else looked great

**Review by marc-gr — APPROVED** [review]

**Pull request merged**